doc/lua_api.txt

```diff
@@ -3345,6 +3345,19 @@ It can be created via `Settings(filename)`.
     * write changes to file
 * `to_table()`: returns `{[key1]=value1,...}`
 
+### `RaycastState`
+A raycast on the map. It works with selection boxes.
+
+It can be created via `RaycastState(pos1, pos2, objects, liquids)` where:
+    * `pos1`: start of the shootline
+    * `pos2`: end of the shootline
+    * `objects` : if false, only nodes will be returned. Default is true.
+    * `liquids' : if false, liquid nodes won't be returned. Defalt is false.
```

Comment:
This seems rather specific.  I think mods should filter this out or it should be something more generic like checking if it's normally pointable rather than just if it's a liquid.